### PR TITLE
Add brand new logo to the project

### PR DESCRIPTION
**Pull request by caneco**
Following your request in the tweet:
http://twitter.com/antonkomarev/status/1179503331565330437

Here it is your brand new logo. Hope you like it!

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +79,8 @@ You will only be able to access this dashboard when `APP_ENV=local` is set in `.
 ## License
 
 - `Laravel Paket` package is open-sourced software licensed under the [MIT License](LICENSE) by [Anton Komarev].
-- `Shipping` logo image licensed under [Creative Commons 3.0](https://creativecommons.org/licenses/by/3.0/us/) by Gan Khoon Lay.
+- `Laravel Paket` logo by [Caneco](http://twitter.com/caneco).
+- Old `Shipping` logo image licensed under [Creative Commons 3.0](https://creativecommons.org/licenses/by/3.0/us/) by Gan Khoon Lay.
```
Comment (antonkomarev) [inline README.md]: No need to keep old license. It will be stored in old versions anyway.

**Comment by antonkomarev** [comment]
Nice one @caneco! But there is a typo in name. Need to remove `c` letter. Can I have SVG version of it to integrate it in web application too?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
```
Comment (antonkomarev) [inline README.md]: Instead of adding it to the top - it could just replace old logo.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+<img width="400" height="130" src="https://raw.githubusercontent.com/cybercog/laravel-paket/master/art/logo.png" alt="Laravel Paket logo" />
```
Comment (antonkomarev) [inline README.md]: Instead of adding it to the top - it could just replace old logo.

**Comment by caneco** [comment]
fix:
- simple gradients
- removed the extra "c" … sorry about the typo

image

**Comment by antonkomarev** [comment]
Yeah! That's much better. Could you send me an SVG of the logo? I want to make small copy of it for the web version.

**Comment by caneco** [comment]
> Yeah! That version is much better.
GREAT!

> Could you send me an SVG of the logo? I want to make small copy of it for the web admin panel.
> Why don't you replace current logo with new one? There is no need to leave old variant.
Because I want to have your opinion beforehand :)

Going to update the PR then…

**Comment by caneco** [comment]
Sorry the delay… just updated the PR 🚀

**Pull request merged**